composer.json

```diff
@@ -31,14 +31,18 @@
         "drush/drush": "^10.4"
     },
     "scripts": {
+        "coding-standards-check/twigcs": [
+            "twigcs web/profiles/custom/os2loop/themes/os2loop_theme/templates"
+        ],
```

Comment:
For consistency this block should be put next to the other coding standards stuff.